services/galley/src/Galley/API/Swagger.hs

```diff
@@ -261,29 +262,17 @@ instance ToSchema ViewLegalHoldServiceInfo where
           Right lhuri = mkHttpsUrl [uri|https://example.com/|]
           fpr = Fingerprint "\138\140\183\EM\226#\129\EOTl\161\183\246\DLE\161\142\220\239&\171\241h|\\GF\172\180O\129\DC1!\159"
 
-instance ToSchema LegalHoldTeamConfig where
-  declareNamedSchema = genericDeclareNamedSchema opts
-    where
-      opts =
-        defaultSchemaOptions
-          { fieldLabelModifier = \case
-              "legalHoldTeamConfigStatus" -> "status"
-          }
-
-instance ToSchema LegalHoldStatus where
-  declareNamedSchema = tweak . genericDeclareNamedSchema opts
+instance ToSchema TeamFeatureStatus where
+  declareNamedSchema _ =
+    pure $ NamedSchema (Just "TeamFeatureStatus") $
+      mempty
+        & properties .~ (fromList [("status", Inline status)])
+        & required .~ ["status"]
+        & type_ ?~ SwaggerObject
     where
-      opts =
-        defaultSchemaOptions
-          { constructorTagModifier = \case
-              "LegalHoldDisabled" -> "disabled"
-              "LegalHoldEnabled" -> "enabled"
-          }
-      tweak = fmap $ schema . description ?~ descr
-        where
-          descr =
-            "determines whether admins of a team "
-              <> "are allowed to enable LH for their users."
+      status =
+        mempty
+          & enum_ ?~ [String "enabled", String "disabled"]
```

Comment:
thanks, this is at least as good as the generic-then-mutate instance that was here before, but can you keep the description?

Reply:
Do you mean we should combine the descriptions for legalhold, sso and search-visibility and add it here?